### DISCUSSION TITLE
[问题]可以使用微信小程序的插件miniprogram-recycle-view吗？使用指导有吗？

### DISCUSSION THREAD

**Discussion by fewoy** [remaxjs/remax#1614] [Q&A]
**问题描述**
目前我有长列表的需求，remax-recycle-view不稳定有bug，想直接使用微信小程序的插件miniprogram-recycle-view。

**示例代码**
https://developers.weixin.qq.com/miniprogram/dev/extended/component-plus/recycle-view.html

**其他信息**
能否提供使用指导？

**Comment by watsonhaw5566** [comment]
remax-recycle-view，这个库你遇到了什么问题？
